epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -421,30 +436,51 @@ int main(int argc, char **argv)
 
     for (int ievt = 0; ievt < nevt; ++ievt) // Loop over all events in this iteration
     {
-      if (verbose)
-      {
-        // Display momenta
-        const int ipagM = ievt/neppM; // #eventpage in this iteration
-        const int ieppM = ievt%neppM; // #event in the current eventpage in this iteration
-        std::cout << "Momenta:" << std::endl;
+      auto getMomentum = [&](std::size_t evtNo, int particle, int component) {
+        assert(component < mgOnGpu::np4);
```

Comment:
I am not used to asserts like this, but you are most probably right :-)
Is this the recommended way for internal consistency checks? (that you can disable with NDEBUG)?

We should ask @oliviermattelaer but I guess he does not mind?